package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
I would like to suggest that `istanbul` be in the `optionalDependencies`, which will allow the user to any `istanbul` compatible replacement like [`babel-istanbul`](/ambitioninc/babel-istanbul) or `gotwarlost/istanbul#source-map`.
Which is what I'm suggesting in #28